BitwardenWatchApp/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CLKComplicationPrincipalClass</key>
```

Comment:
🎨 If you run this through plutil (`plutil -convert xml1 Info.plist`), it will automatically alphabetize the keys for you (in addition to just cleaning up the formatting in general).

Reply:
Awesome nice trick thanks!